ControlLaptop/Examples/Log-windtunnel-influx/log-windtunnel-influx.py

```diff
@@ -0,0 +1,170 @@
+# Example for logging data to influxdb using python
+# Sourced from the influxdb python docs
+
+import random
+import time
+from datetime import datetime 
+from influxdb import InfluxDBClient
```

Comment:
Imports should be ordered alphabetically, they should be as follows:
first-party imports (random, time, anything default to python)

third-party imports (influx, any other packages)

user-defined imports (Anything you wrote and are importing)